source/leadership.rst

```diff
@@ -0,0 +1,133 @@
+.. _leadership:
+
+##################
+Project Leadership 
+##################
+
+yt-project Members
+------------------
+
+A member is someone who has made continued and significant contribution to the
+project (changes to the codebase, discussion on mailing lists, feedback on pull
+requests, documentation, teaching, etc.) for some period of time. After such a
+period, potential new members are nominated for membership by an existing
+member and confirmed by positive votes from three additional members. Once a
+developer becomes a member, they remain a member for life. A member maintains
+the option to give up their membership and have their name removed from the
+list. Membership may be revoked for anyone who is deemed to be directly harmful
+to the project or the community upon a nomination by another member and five
+supporting member votes and an additional vote by the steering committee.
+Members may also elect to become emeritus members, and be recognized for their
+contributions to the project while also rescinding their write access to the
+project. 
+
+Upon the initial creation of yt membership status, all
+developers that had contributed at least 50 changesets were granted
+membership. The `Project Members page <https://yt-project.org/members.html>`_ 
+gives a description of the requirements
+for membership and provides a list of all members and the year in which
+membership was granted. Members have write access to all official yt
```

Comment:
As a note, this does mean that we will increasingly have write access given out.  I think that's something to keep in mind, and perhaps re-evaluate if it becomes an issue.  (It is not at present an issue!)

Reply:
One thing that I was thinking is that maybe we want some sort of tiered write access? These guidelines give people who contribute to the yt core package write access to packages like unyt theoretically, right? Maybe we want to grant write access to the yt core package with these requirements, and then after another set of contributions are made they'd get write access to the project itself?

Reply:
I'm in favor of returning to this issue and thinking about practical reasoning for write access rather than membership.